SR2025/2025-02-24-feb-update.md

```diff
@@ -0,0 +1,56 @@
+---
+to: SR2025 teams
+subject: SR2025 February Update
+---
+
+Hi,
+
+With the competition on the horizon we have a few updates and reminders for you:
+
+* The final [Tech Day][tech-days] of the year is in Cambridge, [on 8th March][cambridge-tech-day-march]
+* The [Competition][programme-competition] is under two months away
+
+## Cambridge Tech Day
+
+[Tech Days][tech-days] are opportunities for teams to spend a whole day working on their robot with lots of help available -- from a mix of long-experienced volunteers and more recent ex-competitors. They're also an opportunity to see how other teams are doing or get more direct help with your robot.
+
+We provide a space for you to work in, with power and internet access, as well as volunteers able to help you with your kits and hands-on guidance with your robot.
+
+There is one final Tech Day left this year, on [**8th March 2025** - at Raspberry Pi Foundation in Cambridge][cambridge-tech-day-march].
+
+There are still plenty of spaces available, so if you're interested in attending please **[sign up][tech-day-signup]**.
+Volunteers may also be around virtually for support, should you not be able to attend the Tech Day in person.
+
+For this Tech Day, the deadline is Friday 28th February 2025 for you to [let us know][tech-day-signup] that you'd like to attend.
+
+## Competition
+
+As you are hopefully aware, the [Competition][programme-competition] is a two day event and is the culmination of all your team's hard work over the past six months.
+This year the competition is being held at the University of Southampton on the weekend of [**Saturday 12th April** and **Sunday 13th April**][competition-event].
+
+Your robot will compete in a number of league matches, used to seed the knockout stages, as well as having time to make improvements.
+You will have a pit area where you can work on improving your robot and there will be lots of Blueshirt volunteers around to support your team.
+
+If you haven't already done so now is a good time to plan your trip, including travel and accommodation for the weekend, and consider any paperwork which your institution may require.
+If you require a risk assessment for the event, please reach out and we will be happy to provide one.
```

Comment:
Think this has been published now, we should just link to it

Reply:
Don't know. Put the link in the email.

Reply:
2087589ebde9af65ec5476eccc776104d7ed052f